scripts/issues.coffee

```diff
@@ -0,0 +1,43 @@
+# Description:
+#   Create GitHub issues for CH beheer and PC.COM. Please be concise.
+#
+# Dependencies:
+#   octonode
+#
+# Configuration:
+#   HUBOT_GITHUB_TOKEN = <accessToken>
+#
+# Commands:
+#   @pccom <query> - Creates a new issue and tags it 'pccom'
+#   @beheer <query> - Creates a new issue and tags it 'beheer'
+
+github = require 'octonode'
+
+githubToken = process.env.HUBOT_GITHUB_TOKEN;
+
+issue = null
```

Comment:
JS is untyped, dus waar haal je `Boolean` vandaan? [Docs](https://github.com/github/hubot/blob/master/docs/scripting.md#custom-listeners) zeggen dat het een "truthy return value" moet zijn, en een object is true.